Client/Frontend/Theme/ThemedWidgets.swift

```diff
@@ -157,3 +158,9 @@ class ThemedTableSectionHeaderFooterView: UITableViewHeaderFooterView, Themeable
     }
 }
 
+class UISwitchThemed: UISwitch {
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        onTintColor = UIColor.theme.general.controlTint
```

Comment:
`layoutSubviews` seemed like a reliable place to pick up the current tint color, I couldn't find a better candidate.